### PR TITLE
added dense and activation generic layers spec

### PR DESCRIPTION
**Pull request by esube**
I have added the generic spec for dense and generic activation layers.
I have also fixed a type in convolution layer from ndarray to tuple as ndarray is too specific.

So, my idea is to make these layers as generic as possible not to be bogged down to very specific layers in each framework or library with the most common (intersection) arguments and pass other specific arguments as kwargs as dictionary. What do you think?

This PR closes issue #5

### CODE REVIEW

**Comment by brollb** [comment]
Added an example of adding an enum type for the activation functions. I added some of the activation functions but I don't see the info in the keras layer info. Do you know if you can get the activation types from parsing keras?

**Comment by esube** [comment]
Oh, I thought those were parsed under advanced activations. I will check them.
In the process I think, conflict is introduced.

File: universal/spec.yml [hunk 1]
```diff
@@ -37,8 +37,45 @@ attrs:
...
+
+# Activation layer
+# This is a generic activation layer that translates in to most of the activation types
+name: Activation
+attrs:
+    - name: activation
+      description: the name of the activation type
+      type: enum
+      values:
+          - HardShrink
```
Comment (brollb) [inline universal/spec.yml:65]: I added some values here from torch as an example basically listing the supported activation functions. Thoughts?

**Comment by esube** [comment]
I am going to merge this and do another PR for other types of advanced activations

**Pull request merged**